src/components/DetailsMenu/DetailsMenuCard/index.tsx

```diff
@@ -17,41 +19,56 @@ interface DetailsMenuProps {
 }
 
 const DetailsMenuCard = forwardRef<HTMLDivElement, DetailsMenuProps>(
-  ({ "data-cy": dataCy }, ref) => (
-    <Container ref={ref} data-cy={dataCy}>
-      <Row>
-        <Column>
-          <SearchRangeInput />
-          <WrapToggle />
-          <CaseSensitiveToggle />
-        </Column>
-        <Column>
-          <FilterLogicToggle />
-          <ExpandableRowsToggle />
-          <PrettyPrintToggle />
-        </Column>
-      </Row>
-      <ButtonRow />
-      <CLIInstructions />
-    </Container>
-  )
+  ({ "data-cy": dataCy }, ref) => {
+    const [selectedTab, setSelectedTab] = useState(0);
+    return (
+      <Container ref={ref} data-cy={dataCy}>
+        <H3>Parsley Settings</H3>
+        <Tabs
+          aria-label="Details Card Tabs"
+          selected={selectedTab}
+          setSelected={setSelectedTab}
+        >
+          <Tab name="Log Structure">
+            <Row>
+              <Column>
+                <SearchRangeInput />
+                <WrapToggle />
```

Comment:
Wrap appears on both tabs (I think it makes more sense only in Log Viewing Preferences)

Reply:
Oops thanks!